libnavigation-core/src/main/java/com/mapbox/navigation/core/adasis/ADASISv2Message.kt

```diff
@@ -0,0 +1,37 @@
+package com.mapbox.navigation.core.adasis
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * Describes a class holding methods for ADASISv2 messages.
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+class ADASISv2Message internal constructor(
```

Comment:
Don't we need that static `from...` methods?

Reply:
I don't think so. There's no use-case when user creates message from json or whatever

Reply:
Can be removed. We will remove on NN's end as well